Add the PetruziaMini Handwired Keyboard (#23201)
qmk/qmk_firmware: keyboards/handwired/petruziamini/readme.md

Reviewer: Does not follow [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).

# handwired/petruziamini

![handwired/petruziamini]

My first handwired projec. My hoping is to learn how to use github and qmk firmware in one shot.

* Keyboard Maintainer: [LLLKST](https://github.com/LLLKST)

> * Hardware Availability: *i will compile this later if it will work. I will also increase my level of english, i promise :)*

Reviewer: > i will compile this later if it will work

You should be compiling locally to ensure your submission is valid and functions as expected.

Author: My mistake due to the translation from Italian. I meant that I would fill the readme file with information later. I compiled the code locally and it works. I have now made the other required changes as well. Thank you

# PetruziaMini

> ![PetruziaMini](https://imgur.com/uk2BSaz)

Reviewer 2: ```suggestion
![PetruziaMini](https://i.imgur.com/uk2BSazh.jpeg)
```

> ![PetruziaMini](imgur.com image replace me!)

Reviewer 2: Please add an image here if possible.

> *My first handwired projec. This is a 36 keys ortholinear thinked to be keymapped as a split kb. My hoping is to learn how to use github and qmk firmware in one shot, Let's try it.*

Reviewer 2: ```suggestion
36 key ortholinear keyboard intended to be mapped as a split keyboard.
```

* Keyboard Maintainer: [LLLKST](https://github.com/LLLKST)
* Hardware Supported: *promicro compatible controller*

> * Hardware Availability: *(https://it.aliexpress.com/item/1005005967686665.html?spm=a2g0o.order_list.order_list_main.36.40f03696oa5yNm&gatewayAdapt=glo2ita)*

Reviewer 2: ```suggestion
* Hardware Availability: [Aliexpress Pro Micro](https://www.aliexpress.us/item/3256805781371913.html)
```

Make example for this keyboard (after setting up your build environment):

>     make petruziamini:default
> 
> Flashing example for this keyboard:
> 
>     make PetruziaMini:default:flash

Reviewer 2: ```suggestion
    make handwired/petruziamini:default

Flashing example for this keyboard:

    make handwired/petruziamini:default:flash
```
